indico/modules/events/editing/operations.py

```diff
@@ -152,17 +153,18 @@ def review_editable_revision(revision, editor, action, comment, tags, files=None
     if action in (EditingReviewAction.update, EditingReviewAction.update_accept):
         _ensure_publishable_files(new_revision)
     revision.editable.revisions.append(new_revision)
+    submitters = revision.editable.contribution.get_manager_list(permission='submit', explicit=False)
```

Comment:
It's unlikely to make a difference most of the time, but wouldn't `explicit=True` be more logical here? If someone is added as a manager to the contribution they are unlikely to be the intended audience for this notification. Also, `explicit=False` is already the default...

```suggestion
    submitters = revision.editable.contribution.get_manager_list(permission='submit', explicit=True)
```

Reply:
oops, yeah, typo